File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    numList = [int(num) for num in s.split(' ')]
    
    min = numList[0]
    max = numList[0]
    
    for num in numList:
        if num < min:
            min = num
        elif num > max:
            max = num

    return f'{min} {max}'

File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.java

import java.util.Arrays;

class Solution {
public String solution(String s) {

String answer = "";
String [] arr = s.split(" ");
int[] result = Arrays.stream(arr).mapToInt(Integer::parseInt).toArray();
Arrays.sort(result);
answer = result[0]+" "+result[arr.length-1];
return answer;
}
}